components/Button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import StyledButton from './Button.styles';
+
+const Button = ({children, url, htmlAttributes }) => (
+  <StyledButton src={url} attributes={htmlAttributes} >{children}</StyledButton>
+);
+
+Button.propTypes = {
+  source: PropTypes.string.isRequired,
```

Comment:
Why is `source` required and what is `source`?

Reply:
this is because I pulled it from another component. I'll review if it is needed but I don't believe so.